### PR TITLE
Show user preview of to be minted piece while making mint

### LINKED ISSUE #418: Show user preview of to be minted piece while making mint
To have a better connection of what people are minting, would be nice to show a little preview of what kind of picture/audio is it while you are writing description &c somewhere around drag&drop field. As filename doesn't need to have much insight for you and with preview user would avoid minting some wrong piece

### To do
- [ ] picture
- [ ] audio
- [ ] gif -- ability to pause it
- [ ] 3D model?
- [ ] make PR
- [ ] post your KSM address

### PR DESCRIPTION
**Pull request by daiagi**
:wrench:  av-waveform component is not working (see image below :point_down: ) :arrow_right:  replaced with native audio tag
:wrench: :  Use MediaResolver in DropUpload component
:wrench:  Render ModelMedia outside of the upload field, in order to prevent click event clash between the field and model-viewer

**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Closes #418 
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [x] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [ ] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=EfmnRhHaQqfT3phm4cUCHCU3gFVDoSPR1U9WXzMRQBMqZ4L)

#### Community participation

- [ ] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

:sound: audio media: 
image

:three:  model media:
image

:bug: av-waveform crash

image

### CODE REVIEW

**Review by Jarsen136 — APPROVED** [review]
code LGTM

File: components/media/type/AudioMedia.vue [hunk 1]
```diff
@@ -1,26 +1,21 @@
...
-    <av-waveform
-      class="media-audio__player"
-      :audio-src="src"
-      audio-class="media-audio__audio"
-      canv-class="media-audio__canvas"
-      played-line-color="rgba(0,0,0,0.74)"
-      noplayed-line-color="#d32e79"
-      playtime-font-color="rgba(0,0,0,0.74)"
+  <div class="is-flex is-justify-content-center is-align-content-center">
+    <audio
```
[the remark this answers is not in the document]
Comment (daiagi) [reply components/media/type/AudioMedia.vue]: reverted

**Comment by daiagi** [comment]
@preschian 
I added one more change in DropUpload.vue
preventing mouse events from bubbling into the form
in order to allow click on the waveform in AudioMedia NOT to trigger file upload menu

**Comment by yangwao** [comment]
seems on mp3 doesn't work? 
tried here https://deploy-preview-4307--koda-nuxt.netlify.app/rmrk/create
image

or do I need to merge here
- #4321 first ?

**Comment by daiagi** [comment]
@yangwao 
correct. need to merge #4321 first.

**Comment by yangwao** [comment]
updated main branch

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $100 @ 25.33 USD/KSM ~ 3.948 $KSM
🧗 [EfmnRhHaQqfT3phm4cUCHCU3gFVDoSPR1U9WXzMRQBMqZ4L](https://kusama.subscan.io/account/EfmnRhHaQqfT3phm4cUCHCU3gFVDoSPR1U9WXzMRQBMqZ4L)
🔗 [0x460c6c1ecf0fb15b97befadbc0cbe7a278188dbe33498abe20b0cd12319bd840](https://kusama.subscan.io/extrinsic/0x460c6c1ecf0fb15b97befadbc0cbe7a278188dbe33498abe20b0cd12319bd840)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Comment by daiagi** [comment]
@yangwao 
Sweet!
Thank you :smile:

**Pull request merged**